### PR TITLE
Import pages through navigation (part 3)

### PR DESCRIPTION
**Pull request by psrpinto**
Follow-up to #114 

This is the last PR of the series, the flow for manual (blueprint-less) import of pages is now functional. We can improve it as needed in future PRs.

## Next steps

- The toolbar needs to be improved, buttons are not aligned, etc. This will be done globally, so that there is a single `<Toolbar/>` component that is used by all screens.

## Screen recording
https://github.com/user-attachments/assets/7c1564d3-8d06-4df6-aaae-054cfac958a6

### CODE REVIEW

**Review by akirk — COMMENTED** [review]
Really nice! Great work and demo! When I tried this locally the body didn't transfer correctly, do you still have local changes not pushed yet?

https://github.com/user-attachments/assets/c16519e8-ad38-4b40-82d9-eecd7686f382

**Comment by ashfame** [comment]
I think it's the same issue I encountered in #110 I suspect paste handler is doing something differently now.

**Comment by psrpinto** [comment]
No local changes, everything is pushed. Could be a difference in npm dependency versions, I'll take a look.

**Comment by akirk** [comment]
I realize I was using `yarn` before, if I use `npm install` it works!

File: src/ui/preview/Playground.tsx [hunk 1]
```diff
@@ -55,7 +55,7 @@ async function initPlayground(
 	slug: string,
 	blogName: string
 ): Promise< PlaygroundClient > {
-	const opfsEnabled = true;
```
Comment (ashfame) [inline src/ui/preview/Playground.tsx:58]: @psrpinto This got committed

Comment (psrpinto) [reply src/ui/preview/Playground.tsx:58]: Oops, undoing. I'll make sure it doesn't happen in the future.

**Pull request merged**